src/client/context/AuthContext.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import * as firebase from "firebase/app";
+
+const AuthContext = React.createContext();
+
+export class Provider extends React.Component {
+  state = {
+    isAuthenticated: false
+  };
+
+  componentDidMount() {
+    firebase.auth().onAuthStateChanged(user => {
+      {
+        user && console.log("User", user.email);
```

Comment:
```suggestion
        // eslint-disable-next-line no-console
        user && console.log("User", user.email);
```

Please add the above comment to bypass the eslint rule that disallow use of console.log. We should generally not be using console.log, but in this case we are waiting for the private routes component to be finished, so it is a viable way to test things in the meantime, but as soon as my PR with ESlint rules is merged we will have errors on console.log statements if you don't explicitly state that this line should be ignored. 

Do the same for other console.log statements in this PR.

Reply:
done